README.md

```diff
@@ -33,75 +33,112 @@ Confused about the terminology of APM? Take a look at the [APM Glossary][visuali
 
 See [the contributing docs](https://ddtrace.readthedocs.io/en/stable/contributing.html) first.
 
-### Pre-commit Hooks
+### Git hooks
 
-The tracer library uses formatting/linting tools including black, flake8, and mypy.
-While these are run in each CI pipeline for pull requests, they are automated to run
-when you call `git commit` as pre-commit hooks to catch any formatting errors before
-you commit. To initialize the pre-commit hook script to run in your development
-branch, run the following command:
+In CI, the [pre_check](pre_check) job ensures consistency in coding style and
+formatting as well as type checking.
 
-    $ hooks/autohook.sh install
+For local development, the repository includes these same checks as custom
+scripts that run as Git pre-commit hooks.
+
+To enable all pre-commit hooks in your cloned repository, run the following command:
 
-### Set up your environment
+    $ hooks/autohook.sh install
+    
+[pre_check]: https://github.com/DataDog/dd-trace-py/blob/5b97489e2b073fa773819904cdef26981a5a28df/.circleci/config.yml#L293-L315
 
-#### Set up docker
+### Development environment
 
-The test suite requires many backing services such as PostgreSQL, MySQL, Redis
-and more. We use `docker` and `docker-compose` to run the services in our CI
-and for development. To run the test matrix, please [install docker][docker] and
-[docker-compose][docker-compose] using the instructions provided by your platform. Then
-launch them through:
+#### Docker
 
-    $ docker-compose up -d
+Install [Docker](docker) on your development environment for running tests
+locally.
 
 [docker]: https://www.docker.com/products/docker
-[docker-compose]: https://www.docker.com/products/docker-compose
 
-#### Set up Python
+#### Python
 
-1. Clone the repository locally: `git clone https://github.com/DataDog/dd-trace-py`
-2. The tests for this project run on various versions of Python. We recommend
-   using a Python version management tool, such as
-   [pyenv](https://github.com/pyenv/pyenv), to utilize multiple versions of
-   Python. Install Pyenv: https://github.com/pyenv/pyenv#installation
-3. Install the relevant versions of Python in Pyenv: `pyenv install 3.9.1, 2.7.18, 3.5.10, 3.6.12, 3.7.9, 3.8.7, 3.10.0`
-4. Make those versions available globally: `pyenv global 3.9.1, 2.7.18, 3.5.10, 3.6.12, 3.7.9, 3.8.7, 3.10.0`
+The library supports multiple Python runtime versions and platforms. See [supported runtimes](https://ddtrace.readthedocs.io/en/stable/versioning.html#supported-runtimes) for the full list. You have two options for preparing a local development environment. You can use pyenv to install Python runtimes on your local machine or you can use the `datadog/dd-trace-py` Docker image which includes all runtimes.
 
-### Testing
+##### Option #1: pyenv
+
+1. [Install pyenv](https://github.com/pyenv/pyenv#getting-pyenv)
+2. Install Python runtimes with pyenv
+3. [Optional] [Install pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv)
```

Comment:
Can this be removed? The minimum requirement is that a user install riot and tox, which handle creating virtual environments.